### PR TITLE
Concat bar layer circle function that uses the vega-lite-api library …

### PR DESCRIPTION
**Pull request by aqmeraamir**
This change resolves concat_bar_layer_circle issue …to generate and return a vega-lite JSON spec

Signed off by Aqmer Aamir <aqmeraamir@gmail.com>

### CODE REVIEW

File: src/concat_bar_layer_circle.vl.js [hunk 1]
```diff
@@ -5,71 +5,50 @@ import * as vl from 'vega-lite-api';
...
-          },
-          "value": "grey"
-        }
-      }
-    }
-  ]
+
+const vl = require('vega-lite-api');
+
+function generateVegaLiteSpec() {
```
Comment (mhkeller) [inline src/concat_bar_layer_circle.vl.js:11]: Thanks for the contribution. Take a look at the structure of some of the already completed ones and you'll see you can save a few steps. Change this to `export default function chart() {` (the name doesn't matter but may help to keep it consistent

File: src/concat_bar_layer_circle.vl.js [hunk 2]
```diff
@@ -5,71 +5,50 @@ import * as vl from 'vega-lite-api';
...
+          vl.x().field('Major Genre').axis({ labelAngle: -40 }),
+          vl.y().aggregate('count'),
+          vl.color().condition(
+            { param: 'pts', value: 'steelblue' },
+            vl.value('grey')
+          )
+        )
+      )
+    )
+    .toJSON();
```
Comment (mhkeller) [inline src/concat_bar_layer_circle.vl.js:50]: Change this to `.toSpec()`

**Comment by aqmeraamir** [comment]
Does that fix it?

**Comment by mhkeller** [comment]
Can you add the comment showing the spec back into the file?

**Comment by mhkeller** [comment]
You have to run `npm test` locally and make sure all of the tests pass.

**Pull request closed without merging**